platforms/openal/SoundSystemAL.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <thirdparty/OpenAL/Include/al.h>
+#include <thirdparty/OpenAL/Include/alc.h>
```

Comment:
There should be an `#ifdef _WIN32` block so it can use normal headers on Linux.

Reply:
Normal headers? I don't understand.

Reply:
Like this:

```c
#ifdef _WIN32
#include <thirdparty/OpenAL/Include/al.h>
#include <thirdparty/OpenAL/Include/alc.h>
#else
#include <AL/al.h>
#include <AL/alc.h>
#endif
```

This way Linux users don't have to manually copy in header files.